nncf/experimental/tensor/functions/torch_numeric.py

```diff
@@ -242,3 +242,34 @@ def _(
 @numeric.size.register(torch.Tensor)
 def _(a: torch.Tensor) -> int:
     return torch.numel(a)
+
+
+@numeric.quantile.register(torch.Tensor)
+def _(
+    a: torch.Tensor,
+    q: float,
```

Comment:
please allow to use several quantiles
```suggestion
    q: Union[float, Tuple[float, ...],
```

Reply:
Done